### ISSUE TITLE
Changelog?

### ISSUE THREAD

**Issue by can3p** [LispCookbook/cl-cookbook#143]
Hey,

I've subscribed to the changes of this repo and that allows to track changes in different chapters. However I don't think that this works for everybody, so if typical person visits the site like once one-two months, there is a little choice other than scanning through all the pages in search for a new content.

What do you think, what's the best way to handle this?

On my projects I prefer to keep a changelog file (e.g. https://can3p.github.io/cl-journal/Changelog.html) which makes tracking easier.

**Comment by pnathan** [comment]
I prefer `git log`.

Other people might prefer otherwise.  I welcome discussion on this matter.

**Comment by can3p** [comment]
Yep, git log works, it's just isolated and I wonder how all the changes can be shared more effectively. Maybe it's not a real problem and can be solved by occasional blog posts here and there

**Comment by vindarel** [comment]
>  occasional blog posts here and there

what if we had a meta-issue called changelog to use as a blog, where someone (the commiter, a maintainer or someone else) can write a new comment to write a changelog entry, whenever he wants (after a big commit, or once a month after small changes) ? That way one can subscribe to only this issue, and not watch the entire repository.

I like the idea and would qualify it as "nice to have", not a "should" as with a software or a library. Also I see a changelog file like duplication of data (with the commit which in our case is clear about its changes) and I'd avoid asking a committer to fill it, or have changelog-only commits.

It could be nice to have a small list of recent changes on the front page, but I'd rather do that with a proper publication software, not our simple gh pages.

Other (new-ish contributors) opinions ?

**Comment by vindarel** [comment]
btw, I'm for a proper changelog now.
